src/actinia_core/version.py

```diff
@@ -133,3 +135,20 @@ def version():
     info['running_since'] = find_running_since_info()
 
     return make_response(jsonify(info), 200)
+
+
+# Return a hint that this version is outdated
+@flask_app.route("/api/v1/<path:actinia_path>")
```

Comment:
It could but usually a proxy is used to direct requests to the same root URL to the corresponding app. If actinia v1 is also installed, the proxy needs to take care about corresponding routing and to overwrite these endpoints.

Do you mean OLD_VERSION as variable in the code? I would argue against it that if we keep it as is, we can still decide in a later change if we want to add an additional endpoint for the newer older version or just return the hint for the latest deprecated version.
If you mean to replace `v1` in the URL with `<old_version>`, I think it is too risky because it provides a duplicate endpoint for all endpoints because it also fits to `v2`. In my current test it did work but I don't know if flask behaves predictable in this case.

Reply:
I mean OLD_VERSION as variable. I do not think that we need vor each old verson the hint, but it is also fine for me to keep it like it is